### PR TITLE
Add credit links

### PR DESCRIPTION
**Pull request by RyanLua**
## Changes

Adds links to where an Roblox experience is mentioned so you can easily access the experiences without having to search it up.

## Checks

By submitting your pull request for review, you agree to the following:

- [x] This contribution was created in whole or in part by me, and I have the right to submit it under the terms of this repository's open source licenses.
- [x] I understand and agree that this contribution and a record of it are public, maintained indefinitely, and may be redistributed under the terms of this repository's open source licenses.
- [x] To the best of my knowledge, all proposed changes are accurate.

### CODE REVIEW

File: content/en-us/production/game-design/content-updates.md [hunk 1]
```diff
@@ -28,16 +28,16 @@ A content cadence draws on existing gameplay systems to produce small, frequent
...
 - Quests
 
-Many of these examples are predominantly art-based, requiring little to no programming or game design work to produce. Simple variants on existing assets, like assets with only slight changes to color, are ideal because they can be produced quickly on a recurring schedule by the fewest possible team members. A good example of a content cadence are Pet Variants in _Adopt Me!_
+Many of these examples are predominantly art-based, requiring little to no programming or game design work to produce. Simple variants on existing assets, like assets with only slight changes to color, are ideal because they can be produced quickly on a recurring schedule by the fewest possible team members. A good example of a content cadence are Pet Variants in [_Adopt Me!_](https://www.roblox.com/games/920587237/)
 
 <GridContainer numColumns="2">
   <figure>
     <img src="../../assets/game-design/content-update/content-update2.png" width="500%"/>
-    <figcaption>Spotted pet variant in *Adopt Me!*</figcaption>
+    <figcaption>Spotted pet variant in [*Adopt Me!*](https://www.roblox.com/games/920587237/)</figcaption>
```
Comment (aetter) [inline content/en-us/production/game-design/content-updates.md:36]:
```suggestion
    <figcaption>Spotted pet variant in *Adopt Me!*</figcaption>
```

File: content/en-us/production/game-design/introduction-to-quest-design.md [hunk 2]
```diff
@@ -150,7 +150,7 @@ Quests can encourage players to play the game in more interesting and challengin
 
   <figure>
       <img src="../../assets/game-design/introduction-to-quest-design/quest-design-4.png" width="70%"/>
-      <figcaption>Achievement badge in *Jailbreak*</figcaption>
+      <figcaption>Achievement badge in [*Jailbreak*](https://www.roblox.com/games/606849621/)</figcaption>
```
Comment (aetter) [inline content/en-us/production/game-design/introduction-to-quest-design.md:153]:
```suggestion
      <figcaption>Achievement badge in *Jailbreak*</figcaption>
```

File: content/en-us/production/game-design/monetization-foundations.md [hunk 3]
```diff
@@ -91,21 +91,21 @@ To be effective, a shop needs to be effectively integrated within the experience
...
 
 - **Future anticipation:** Players are alerted that "Knobs", the experience's in-game currency, is receiving more use cases in the future. This alerts players that additional content is coming to utilize the in-game currency, which both builds anticipation for a future release and increases the perceived value of the product.
 
 - **Present explanation:** Players are given the explanation and context of purchasable "Revives". This alerts players that reviving is a core part of the gameplay experience and that this functionality is limited. As players navigate the shop, explanations like this help players accumulate knowledge about the experience.
   <figure>
     <img src="../../assets/game-design/monetization-foundations/monetization-foundation-4.png" width="70%"/>
-    <figcaption>Shop in *Doors*</figcaption>
+    <figcaption>Shop in [*Doors*](https://www.roblox.com/games/6516141723/)</figcaption>
```
Comment (aetter) [inline content/en-us/production/game-design/monetization-foundations.md:108]:
```suggestion
    <figcaption>Shop in *Doors*</figcaption>
```

File: content/en-us/production/game-design/monetization-foundations.md [hunk 4]
```diff
@@ -156,31 +156,31 @@ You can effectively communicate the value of items in your shop by employing the
...
 <GridContainer numColumns="2">
   <figure>
     <img src="../../assets/game-design/monetization-foundations/monetization-foundation-8.png" width="90%"/>
-    <figcaption>Shop in *Car Dealership Tycoon*</figcaption>
+    <figcaption>Shop in [*Car Dealership Tycoon*](https://www.roblox.com/games/1554960397/)</figcaption>
   </figure>
   <figure>
     <img src="../../assets/game-design/monetization-foundations/monetization-foundation-9.png" width="90%"/>
-    <figcaption>Shop in _The Survival Game_</figcaption>
+    <figcaption>Shop in [_The Survival Game_](https://www.roblox.com/games/11156779721/)</figcaption>
```
Comment (aetter) [inline content/en-us/production/game-design/monetization-foundations.md:175]:
```suggestion
    <figcaption>Shop in _The Survival Game_</figcaption>
```

File: content/en-us/production/game-design/monetization-foundations.md [hunk 5]
```diff
@@ -193,18 +193,18 @@ Another common merchandising strategy is to use a chance-based system. In chance
 
   <figure>
       <img src="../../assets/game-design/monetization-foundations/monetization-foundation-11.png" width="70%"/>
-      <figcaption>Chance wheel in *Dragon Adventure*</figcaption>
+      <figcaption>Chance wheel in [*Dragon Adventure*](https://www.roblox.com/games/3475397644/)</figcaption>
```
Comment (aetter) [inline content/en-us/production/game-design/monetization-foundations.md:196]:
```suggestion
      <figcaption>Chance wheel in *Dragon Adventure*</figcaption>
```

File: content/en-us/production/game-design/monetization-foundations.md [hunk 6]
```diff
@@ -325,7 +325,7 @@ A good season pass is contextual and organized, clearly communicating the reward
 
 <figure>
     <img src="../../assets/game-design/monetization-foundations/monetization-foundation-19.png" width="70%"/>
-    <figcaption>Season pass in *Bot Clash Simulator*</figcaption>
+    <figcaption>Season pass in [*Bot Clash Simulator*](https://www.roblox.com/games/9300407930/)</figcaption>
```
Comment (aetter) [inline content/en-us/production/game-design/monetization-foundations.md:328]:
```suggestion
    <figcaption>Season pass in *Bot Clash Simulator*</figcaption>
```

File: content/en-us/production/game-design/monetization-foundations.md [hunk 7]
```diff
@@ -334,7 +334,7 @@ Good season passes are inviting. As a subset of an experience's shop, season pas
 
 <figure>
     <img src="../../assets/game-design/monetization-foundations/monetization-foundation-20.png" width="70%"/>
-    <figcaption>Descriptive and enticing rewards in *Bot Clash Simulator*</figcaption>
+    <figcaption>Descriptive and enticing rewards in [*Bot Clash Simulator*](https://www.roblox.com/games/9300407930/)</figcaption>
```
Comment (aetter) [inline content/en-us/production/game-design/monetization-foundations.md:337]:
```suggestion
    <figcaption>Descriptive and enticing rewards in *Bot Clash Simulator*</figcaption>
```

File: content/en-us/production/game-design/onboarding.md [hunk 8]
```diff
@@ -108,16 +108,16 @@ Traditional tutorial flows guide players in a fixed sequence through an experien
...
 <h4>Use Cases</h4>
 
-In _Squishmallows_, contextual tutorials are used to effectively teach players how and when to use the combination station. When a player has two identical Squishmallows in their inventory, a contextual tutorial triggers, and shows players how to combine three _Squishmallows_ at the combination station in order to create a larger one.
+In [_Squishmallows_](https://www.roblox.com/games/7941853407/), contextual tutorials are used to effectively teach players how and when to use the combination station. When a player has two identical Squishmallows in their inventory, a contextual tutorial triggers, and shows players how to combine three _Squishmallows_ at the combination station in order to create a larger one.
 
 <GridContainer numColumns="2">
   <figure>
     <img src="../../assets/game-design/onboarding/onboarding_5.png" width="100%"/>
-    <figcaption>Contextual combination station tutorial in *Squishmallows*.</figcaption>
+    <figcaption>Contextual combination station tutorial in [*Squishmallows*](https://www.roblox.com/games/7941853407/).</figcaption>
```
Comment (aetter) [inline content/en-us/production/game-design/onboarding.md:116]:
```suggestion
    <figcaption>Contextual combination station tutorial in *Squishmallows*.</figcaption>
```

File: content/en-us/production/game-design/onboarding.md [hunk 9]
```diff
@@ -108,16 +108,16 @@ Traditional tutorial flows guide players in a fixed sequence through an experien
...
   </figure>
   <figure>
     <img src="../../assets/game-design/onboarding/onboarding_6.png" width="100%"/>
-    <figcaption>Contextual combination station tutorial in *Squishmallows*.</figcaption>
+    <figcaption>Contextual combination station tutorial in [*Squishmallows*](https://www.roblox.com/games/7941853407/).</figcaption>
```
Comment (aetter) [inline content/en-us/production/game-design/onboarding.md:120]:
```suggestion
    <figcaption>Contextual combination station tutorial in *Squishmallows*.</figcaption>
```

**Comment by aetter** [comment]
Hey @RyanLua, I love the idea here, but I'd rather just link on first mention on each page rather than repeatedly. I'll start applying suggestions so it's a bit easier to review.

File: content/en-us/production/game-design/onboarding.md [hunk 10]
```diff
@@ -128,10 +128,10 @@ Contextual tutorials also teach non-core concepts, letting players learn about t
 <GridContainer numColumns="2">
   <figure>
     <img src="../../assets/game-design/onboarding/onboarding_7.png" width="100%"/>
-    <figcaption>Contextual market tutorial in *Squishmallows*.</figcaption>
+    <figcaption>Contextual market tutorial in [*Squishmallows*](https://www.roblox.com/games/7941853407/).</figcaption>
```
Comment (aetter) [inline content/en-us/production/game-design/onboarding.md:131]:
```suggestion
    <figcaption>Contextual market tutorial in *Squishmallows*.</figcaption>
```

File: content/en-us/production/game-design/onboarding.md [hunk 11]
```diff
@@ -128,10 +128,10 @@ Contextual tutorials also teach non-core concepts, letting players learn about t
...
   </figure>
   <figure>
     <img src="../../assets/game-design/onboarding/onboarding_8.png" width="100%"/>
-    <figcaption>Contextual market tutorial in *Squishmallows*.</figcaption>
+    <figcaption>Contextual market tutorial in [*Squishmallows*](https://www.roblox.com/games/7941853407/).</figcaption>
```
Comment (aetter) [inline content/en-us/production/game-design/onboarding.md:135]:
```suggestion
    <figcaption>Contextual market tutorial in *Squishmallows*.</figcaption>
```

File: content/en-us/production/game-design/season-pass-design.md [hunk 12]
```diff
@@ -18,7 +18,7 @@ Season passes are time-bound design strategies used in Roblox experiences to del
 
 <figure>
     <img src="../../assets/game-design/season-pass-design/season-pass-design-0.png" width="70%"/>
-    <figcaption>Season Pass in *Jailbreak*</figcaption>
+    <figcaption>Season Pass in [*Jailbreak*](https://www.roblox.com/games/606849621/)</figcaption>
```
Comment (aetter) [inline content/en-us/production/game-design/season-pass-design.md:21]:
```suggestion
    <figcaption>Season Pass in *Jailbreak*</figcaption>
```

File: content/en-us/production/game-design/season-pass-design.md [hunk 13]
```diff
@@ -74,7 +74,7 @@ The _Season length_ is the duration that a season of rewards is available to pla
 
 <figure>
     <img src="../../assets/game-design/season-pass-design/season-pass-design-4.png" width="70%"/>
-    <figcaption>Season Length in *Jailbreak*</figcaption>
+    <figcaption>Season Length in [*Jailbreak*](https://www.roblox.com/games/606849621/)</figcaption>
```
Comment (aetter) [inline content/en-us/production/game-design/season-pass-design.md:77]:
```suggestion
    <figcaption>Season Length in *Jailbreak*</figcaption>
```

File: content/en-us/production/game-design/season-pass-design.md [hunk 14]
```diff
@@ -83,7 +83,7 @@ _Season tiers_ are the milestones that players meet in order to earn rewards thr
 
 <figure>
     <img src="../../assets/game-design/season-pass-design/season-pass-design-5.png" width="70%"/>
-    <figcaption>Season Tiers in *Jailbreak*</figcaption>
+    <figcaption>Season Tiers in [*Jailbreak*](https://www.roblox.com/games/606849621/)</figcaption>
```
Comment (aetter) [inline content/en-us/production/game-design/season-pass-design.md:86]:
```suggestion
    <figcaption>Season Tiers in *Jailbreak*</figcaption>
```

**Comment by RyanLua** [comment]
> Hey @RyanLua, I love the idea here, but I'd rather just link on first mention on each page rather than repeatedly. I'll start applying suggestions so it's a bit easier to review.

Yeah, that seems fine by me to only link it once.

File: content/en-us/production/game-design/season-pass-design.md [hunk 15]
```diff
@@ -104,7 +104,7 @@ Extra attention is needed in implementing the following sections of a season pas
 
 <figure>
     <img src="../../assets/game-design/season-pass-design/season-pass-design-6.png" width="70%"/>
-    <figcaption>Tier Rewards in *Jailbreak*</figcaption>
+    <figcaption>Tier Rewards in [*Jailbreak*](https://www.roblox.com/games/606849621/)</figcaption>
```
Comment (aetter) [inline content/en-us/production/game-design/season-pass-design.md:107]:
```suggestion
    <figcaption>Tier Rewards in *Jailbreak*</figcaption>
```

File: content/en-us/production/game-design/season-pass-design.md [hunk 16]
```diff
@@ -116,11 +116,11 @@ Although your Tiers UI shows the rewards that players earn, it's good practice t
 <GridContainer numColumns="2">
   <figure>
    <img src="../../assets/game-design/season-pass-design/season-pass-design-2.png" width="95%"/>
-    <figcaption> Final reward in *Jailbreak* </figcaption>
+    <figcaption> Final reward in [*Jailbreak*](https://www.roblox.com/games/606849621/) </figcaption>
```
Comment (aetter) [inline content/en-us/production/game-design/season-pass-design.md:119]:
```suggestion
    <figcaption> Final reward in *Jailbreak* </figcaption>
```

File: content/en-us/production/game-design/season-pass-design.md [hunk 17]
```diff
@@ -139,7 +139,7 @@ A _free pass_ is a version of a season pass that is available to all players. Pl
 
 <figure>
     <img src="../../assets/game-design/season-pass-design/season-pass-design-7.png" width="70%"/>
-    <figcaption>Free Season Pass in *Jailbreak*</figcaption>
+    <figcaption>Free Season Pass in [*Jailbreak*](https://www.roblox.com/games/606849621/)</figcaption>
```
Comment (aetter) [inline content/en-us/production/game-design/season-pass-design.md:142]:
```suggestion
    <figcaption>Free Season Pass in *Jailbreak*</figcaption>
```

File: content/en-us/production/game-design/season-pass-design.md [hunk 18]
```diff
@@ -150,7 +150,7 @@ A _premium pass_ is a variation of a season pass that players have to pay in ord
 
 <figure>
     <img src="../../assets/game-design/season-pass-design/season-pass-design-8.png" width="70%"/>
-    <figcaption>Premium season pass in *Jailbreak*</figcaption>
+    <figcaption>Premium season pass in [*Jailbreak*](https://www.roblox.com/games/606849621/)</figcaption>
```
Comment (aetter) [inline content/en-us/production/game-design/season-pass-design.md:153]:
```suggestion
    <figcaption>Premium season pass in *Jailbreak*</figcaption>
```

File: content/en-us/production/game-design/season-pass-design.md [hunk 19]
```diff
@@ -191,11 +191,11 @@ Deriving _mission categories_ from the different activities in your experience i
 <GridContainer numColumns="2">
   <figure>
    <img src="../../assets/game-design/season-pass-design/season-pass-design-13.png" width="100%"/>
-    <figcaption> Daily Missions from *Jailbreak* </figcaption>
+    <figcaption> Daily Missions from [*Jailbreak*](https://www.roblox.com/games/606849621/) </figcaption>
```
Comment (aetter) [inline content/en-us/production/game-design/season-pass-design.md:194]:
```suggestion
    <figcaption> Daily Missions from *Jailbreak* </figcaption>
```

File: content/en-us/production/game-design/season-pass-design.md [hunk 20]
```diff
@@ -191,11 +191,11 @@ Deriving _mission categories_ from the different activities in your experience i
...
   </figure>
   <figure>
     <img src="../../assets/game-design/season-pass-design/season-pass-design-14.png" width="96%"/>
-    <figcaption> Daily Missions from *Dragon Adventures* </figcaption>
+    <figcaption> Daily Missions from [*Dragon Adventures*](https://www.roblox.com/games/3475397644/) </figcaption>
```
Comment (aetter) [inline content/en-us/production/game-design/season-pass-design.md:198]:
```suggestion
    <figcaption> Daily Missions from *Dragon Adventures* </figcaption>
```

File: content/en-us/production/game-design/season-pass-design.md [hunk 21]
```diff
@@ -214,11 +214,11 @@ Like categories, you can designate easy, medium and hard difficulties for tasks
 <GridContainer numColumns="2">
   <figure>
    <img src="../../assets/game-design/season-pass-design/season-pass-design-9.png" width="100%"/>
-    <figcaption> Daily Missions from *Dragon Adventures* </figcaption>
+    <figcaption> Daily Missions from [*Dragon Adventures*](https://www.roblox.com/games/3475397644/) </figcaption>
```
Comment (aetter) [inline content/en-us/production/game-design/season-pass-design.md:217]:
```suggestion
    <figcaption> Daily Missions from *Dragon Adventures* </figcaption>
```

File: content/en-us/production/game-design/season-pass-design.md [hunk 22]
```diff
@@ -214,11 +214,11 @@ Like categories, you can designate easy, medium and hard difficulties for tasks
...
   </figure>
   <figure>
     <img src="../../assets/game-design/season-pass-design/season-pass-design-10.png" width="96%"/>
-    <figcaption> Weekly Missions from *Dragon Adventures* </figcaption>
+    <figcaption> Weekly Missions from [*Dragon Adventures*](https://www.roblox.com/games/3475397644/) </figcaption>
```
Comment (aetter) [inline content/en-us/production/game-design/season-pass-design.md:221]:
```suggestion
    <figcaption> Weekly Missions from *Dragon Adventures* </figcaption>
```

File: content/en-us/production/game-design/season-pass-design.md [hunk 23]
```diff
@@ -230,7 +230,7 @@ Difficulty designations also help you determine the appropriate amount of XP to
 
  <figure>
     <img src="../../assets/game-design/season-pass-design/season-pass-design-12.png" width="60%"/>
-    <figcaption> Season Pass HUD surfacing in *Dragon Adventures* </figcaption>
+    <figcaption> Season Pass HUD surfacing in [*Dragon Adventures*](https://www.roblox.com/games/3475397644/) </figcaption>
```
Comment (aetter) [inline content/en-us/production/game-design/season-pass-design.md:233]:
```suggestion
    <figcaption> Season Pass HUD surfacing in *Dragon Adventures* </figcaption>
```

File: content/en-us/production/game-design/ui-ux-design.md [hunk 24]
```diff
@@ -34,37 +34,37 @@ Important information and features change during the course of a game, depending
...
 <h4>Examples</h4>
 
-Irrelevant elements can cause confusion and distraction. _Spellbound RPG_ has a clean, minimalist UI is particularly crucial for mobile interfaces, as small screens can easily get overwhelmed with excessive buttons, screens, and text.
+Irrelevant elements can cause confusion and distraction. [_Spellbound RPG_](https://www.roblox.com/games/6468323505/) has a clean, minimalist UI is particularly crucial for mobile interfaces, as small screens can easily get overwhelmed with excessive buttons, screens, and text.
 
 <GridContainer numColumns="2">
   <figure>
     <img src="../../assets/game-design/ui-ux-design/ui-ux-design-3.png" width="100%"/>
-    <figcaption>Items prior to selection in *Spellbound RPG*</figcaption>
+    <figcaption>Items prior to selection in [*Spellbound RPG*](https://www.roblox.com/games/6468323505/)</figcaption>
```
Comment (aetter) [inline content/en-us/production/game-design/ui-ux-design.md:42]:
```suggestion
    <figcaption>Items prior to selection in *Spellbound RPG*</figcaption></figcaption>
```

File: content/en-us/production/game-design/ui-ux-design.md [hunk 25]
```diff
@@ -34,37 +34,37 @@ Important information and features change during the course of a game, depending
...
   </figure>
   <figure>
     <img src="../../assets/game-design/ui-ux-design/ui-ux-design-2.png" width="90%"/>
-    <figcaption>Items after selection in *Spellbound RPG*</figcaption>
+    <figcaption>Items after selection in [*Spellbound RPG*](https://www.roblox.com/games/6468323505/)</figcaption>
```
Comment (aetter) [inline content/en-us/production/game-design/ui-ux-design.md:46]:
```suggestion
    <figcaption>Items after selection in *Spellbound RPG*</figcaption></figcaption>
```

File: content/en-us/production/game-design/ui-ux-design.md [hunk 26]
```diff
@@ -34,37 +34,37 @@ Important information and features change during the course of a game, depending
...
 </GridContainer>
 
-In _Super Striker League_, player interactions vary based on whether they control the ball. Without the ball, players can Sprint and Tackle to regain control. Once in possession, the buttons switch to Deke and Pass, enabling players to dodge opponents and pass the ball. By alternating these buttons, the UI stays contextually useful, avoiding unnecessary distractions.
+In [_Super Striker League_](https://www.roblox.com/games/3360853050/), player interactions vary based on whether they control the ball. Without the ball, players can Sprint and Tackle to regain control. Once in possession, the buttons switch to Deke and Pass, enabling players to dodge opponents and pass the ball. By alternating these buttons, the UI stays contextually useful, avoiding unnecessary distractions.
 
 <GridContainer numColumns="2">
   <figure>
     <img src="../../assets/game-design/ui-ux-design/ui-ux-design-6.png" width="100%"/>
-    <figcaption>Contextual buttons in _Super Striker League_</figcaption>
+    <figcaption>Contextual buttons in [_Super Striker League_](https://www.roblox.com/games/3360853050/)</figcaption>
```
Comment (aetter) [inline content/en-us/production/game-design/ui-ux-design.md:55]:
```suggestion
    <figcaption>Contextual buttons in _Super Striker League_</figcaption></figcaption>
```

File: content/en-us/production/game-design/ui-ux-design.md [hunk 27]
```diff
@@ -34,37 +34,37 @@ Important information and features change during the course of a game, depending
...
   </figure>
   <figure>
     <img src="../../assets/game-design/ui-ux-design/ui-ux-design-7.png" width="100%"/>
-    <figcaption>Contextual buttons in _Super Striker League_</figcaption>
+    <figcaption>Contextual buttons in [_Super Striker League_](https://www.roblox.com/games/3360853050/)</figcaption>
```
Comment (aetter) [inline content/en-us/production/game-design/ui-ux-design.md:59]:
```suggestion
    <figcaption>Contextual buttons in _Super Striker League_</figcaption>
```

File: content/en-us/production/game-design/ui-ux-design.md [hunk 28]
```diff
@@ -34,37 +34,37 @@ Important information and features change during the course of a game, depending
...
   </figure>
 </GridContainer>
 
-In _The Survival Game,_ players need to be aware of their health and hunger status, and have quick access to their map, inventory, and tools. This information allows them to make decisions and prioritize actions for survival. If this information or tools are hard to find or missing, survival becomes more challenging and needlessly frustrating.
+In [_The Survival Game,_](https://www.roblox.com/games/11156779721/) players need to be aware of their health and hunger status, and have quick access to their map, inventory, and tools. This information allows them to make decisions and prioritize actions for survival. If this information or tools are hard to find or missing, survival becomes more challenging and needlessly frustrating.
 
 <figure>
     <img src="../../assets/game-design/ui-ux-design/ui-ux-design-1.png" width="60%"/>
-    <figcaption> HUD features in _The Survival Game_ </figcaption>
+    <figcaption> HUD features in [_The Survival Game_](https://www.roblox.com/games/11156779721/) </figcaption>
```
Comment (aetter) [inline content/en-us/production/game-design/ui-ux-design.md:67]:
```suggestion
    <figcaption> HUD features in _The Survival Game_ </figcaption></figcaption>
```

File: content/en-us/production/game-design/ui-ux-design.md [hunk 29]
```diff
@@ -81,25 +81,25 @@ Moderation is key with these tools. Excessive use of bright, moving elements mig
 
 <h4>Examples</h4>
 
-In _Jailbreak_'s season pass UI, a stripe of golden yellow lies beneath the season pass-exclusive icons, drawing attention to the premium rewards and making them visually distinct from the free ones.
+In [_Jailbreak_](https://www.roblox.com/games/606849621/)'s season pass UI, a stripe of golden yellow lies beneath the season pass-exclusive icons, drawing attention to the premium rewards and making them visually distinct from the free ones.
 
 <figure>
     <img src="../../assets/game-design/ui-ux-design/ui-ux-design-8.png" width="60%"/>
-    <figcaption> Season pass UI in _Jailbreak_ </figcaption>
+    <figcaption> Season pass UI in [_Jailbreak_](https://www.roblox.com/games/606849621/) </figcaption>
```
Comment (aetter) [inline content/en-us/production/game-design/ui-ux-design.md:88]:
```suggestion
    <figcaption> Season pass UI in _Jailbreak_ </figcaption>
```

File: content/en-us/production/game-design/ui-ux-design.md [hunk 30]
```diff
@@ -81,25 +81,25 @@ Moderation is key with these tools. Excessive use of bright, moving elements mig
...
   </figure>
 
-In _Dragon Adventures,_ visual priority is given to the highest-value coin bundle by increasing it's size and separating it with more padding, to distinguish it from smaller bundles.
+In [_Dragon Adventures,_](https://www.roblox.com/games/3475397644/) visual priority is given to the highest-value coin bundle by increasing it's size and separating it with more padding, to distinguish it from smaller bundles.
 
 <figure>
     <img src="../../assets/game-design/ui-ux-design/ui-ux-design-9.png" width="60%"/>
-    <figcaption> Coin bundles in _Dragon Adventures_</figcaption>
+    <figcaption> Coin bundles in [_Dragon Adventures_](https://www.roblox.com/games/3475397644/)</figcaption>
```
Comment (aetter) [inline content/en-us/production/game-design/ui-ux-design.md:95]:
```suggestion
    <figcaption> Coin bundles in _Dragon Adventures_</figcaption>
```

File: content/en-us/production/game-design/ui-ux-design.md [hunk 31]
```diff
@@ -81,25 +81,25 @@ Moderation is key with these tools. Excessive use of bright, moving elements mig
...
   </figure>
 
-In _Tower Defense Simulator_, both size and proximity are used to visually group and distinguish Daily Tower Skins from Daily Crates.
+In [_Tower Defense Simulator_](https://www.roblox.com/games/3260590327/), both size and proximity are used to visually group and distinguish Daily Tower Skins from Daily Crates.
 
 <figure>
     <img src="../../assets/game-design/ui-ux-design/ui-ux-design-10.png" width="60%"/>
-    <figcaption> _Tower Defense Simulator_ shop</figcaption>
+    <figcaption> [_Tower Defense Simulator_](https://www.roblox.com/games/3260590327/) shop</figcaption>
```
Comment (aetter) [inline content/en-us/production/game-design/ui-ux-design.md:102]:
```suggestion
    <figcaption> _Tower Defense Simulator_ shop</figcaption>
```

File: content/en-us/production/game-design/ui-ux-design.md [hunk 32]
```diff
@@ -130,7 +130,7 @@ Headers and titles containing significant high-level information should be large
 
 <figure>
     <img src="../../assets/game-design/ui-ux-design/ui-ux-design-13.png" width="40%"/>
-    <figcaption> Clothing Shop in *Winds of Fortune*</figcaption>
+    <figcaption> Clothing Shop in [*Winds of Fortune*](https://www.roblox.com/games/7714627632/)</figcaption>
```
Comment (aetter) [inline content/en-us/production/game-design/ui-ux-design.md:133]:
```suggestion
    <figcaption> Clothing Shop in *Winds of Fortune*</figcaption>
```

File: content/en-us/production/game-design/ui-ux-design.md [hunk 33]
```diff
@@ -148,11 +148,11 @@ Given their widespread use, players are typically familiar with such conventions
 <GridContainer numColumns="2">
   <figure>
     <img src="../../assets/game-design/ui-ux-design/ui-ux-design-14.png" width="100%"/>
-    <figcaption>X Close Buttons in *Winds of Fortune*, *Dragon Adventures*, *BotClashSimulator*, and *DOORS*</figcaption>
+    <figcaption>X Close Buttons in [*Winds of Fortune*](https://www.roblox.com/games/7714627632/), [*Dragon Adventures*](https://www.roblox.com/games/3475397644/), [*BotClashSimulator*](https://www.roblox.com/games/9300407930/), and [*DOORS*](https://www.roblox.com/games/6516141723/)</figcaption>
```
Comment (aetter) [inline content/en-us/production/game-design/ui-ux-design.md:151]:
```suggestion
    <figcaption>X Close Buttons in *Winds of Fortune*, *Dragon Adventures*, *BotClashSimulator*, and *DOORS*</figcaption>
```

File: content/en-us/production/game-design/ui-ux-design.md [hunk 34]
```diff
@@ -148,11 +148,11 @@ Given their widespread use, players are typically familiar with such conventions
...
   </figure>
   <figure>
    <img src="../../assets/game-design/ui-ux-design/ui-ux-design-15.png" width="90%"/>
-    <figcaption>Green "Health" stat consistency in *Winds of Fortune*</figcaption>
+    <figcaption>Green "Health" stat consistency in [*Winds of Fortune*](https://www.roblox.com/games/7714627632/)</figcaption>
```
Comment (aetter) [inline content/en-us/production/game-design/ui-ux-design.md:155]:
```suggestion
    <figcaption>Green "Health" stat consistency in *Winds of Fortune*</figcaption>
```

File: content/en-us/production/game-design/ui-ux-design.md [hunk 35]
```diff
@@ -247,7 +247,7 @@ For example, to equip an item on an avatar in Berry Avenue RP, a player must:
 
  <figure>
     <video controls src="../../assets/game-design/ui-ux-design/ui-ux-design-18-video.mp4"></video>
-    <figcaption>Avatar customization flow in *Berry Avenue RP*.</figcaption>
+    <figcaption>Avatar customization flow in [*Berry Avenue RP*](https://www.roblox.com/games/8481844229/).</figcaption>
```
Comment (aetter) [inline content/en-us/production/game-design/ui-ux-design.md:250]:
```suggestion
    <figcaption>Avatar customization flow in *Berry Avenue RP*.</figcaption>
```

File: content/en-us/production/game-design/ui-ux-design.md [hunk 36]
```diff
@@ -34,37 +34,37 @@ Important information and features change during the course of a game, depending
...
 </GridContainer>
 
-In _Super Striker League_, player interactions vary based on whether they control the ball. Without the ball, players can Sprint and Tackle to regain control. Once in possession, the buttons switch to Deke and Pass, enabling players to dodge opponents and pass the ball. By alternating these buttons, the UI stays contextually useful, avoiding unnecessary distractions.
+In [_Super Striker League_](https://www.roblox.com/games/3360853050/), player interactions vary based on whether they control the ball. Without the ball, players can Sprint and Tackle to regain control. Once in possession, the buttons switch to Deke and Pass, enabling players to dodge opponents and pass the ball. By alternating these buttons, the UI stays contextually useful, avoiding unnecessary distractions.
 
 <GridContainer numColumns="2">
   <figure>
     <img src="../../assets/game-design/ui-ux-design/ui-ux-design-6.png" width="100%"/>
-    <figcaption>Contextual buttons in _Super Striker League_</figcaption>
+    <figcaption>Contextual buttons in _Super Striker League_</figcaption></figcaption>
```
Comment (aetter) [inline content/en-us/production/game-design/ui-ux-design.md:55]:
```suggestion
    <figcaption>Contextual buttons in _Super Striker League_</figcaption>
```

File: content/en-us/production/game-design/ui-ux-design.md [hunk 37]
```diff
@@ -60,11 +60,11 @@ In _Super Striker League_, player interactions vary based on whether they contro
   </figure>
 </GridContainer>
 
-In _The Survival Game,_ players need to be aware of their health and hunger status, and have quick access to their map, inventory, and tools. This information allows them to make decisions and prioritize actions for survival. If this information or tools are hard to find or missing, survival becomes more challenging and needlessly frustrating.
+In [_The Survival Game,_](https://www.roblox.com/games/11156779721/) players need to be aware of their health and hunger status, and have quick access to their map, inventory, and tools. This information allows them to make decisions and prioritize actions for survival. If this information or tools are hard to find or missing, survival becomes more challenging and needlessly frustrating.
 
 <figure>
     <img src="../../assets/game-design/ui-ux-design/ui-ux-design-1.png" width="60%"/>
-    <figcaption> HUD features in _The Survival Game_ </figcaption>
+    <figcaption> HUD features in _The Survival Game_ </figcaption></figcaption>
```
Comment (aetter) [inline content/en-us/production/game-design/ui-ux-design.md:67]:
```suggestion
    <figcaption> HUD features in _The Survival Game_ </figcaption>
```

File: content/en-us/production/game-design/ui-ux-design.md [hunk 38]
```diff
@@ -34,20 +34,20 @@ Important information and features change during the course of a game, depending
...
 <GridContainer numColumns="2">
   <figure>
     <img src="../../assets/game-design/ui-ux-design/ui-ux-design-3.png" width="100%"/>
-    <figcaption>Items prior to selection in *Spellbound RPG*</figcaption>
+    <figcaption>Items prior to selection in *Spellbound RPG*</figcaption></figcaption>
   </figure>
   <figure>
     <img src="../../assets/game-design/ui-ux-design/ui-ux-design-2.png" width="90%"/>
-    <figcaption>Items after selection in *Spellbound RPG*</figcaption>
+    <figcaption>Items after selection in *Spellbound RPG*</figcaption></figcaption>
```
Comment (aetter) [inline content/en-us/production/game-design/ui-ux-design.md:46]:
```suggestion
    <figcaption>Items after selection in *Spellbound RPG*</figcaption>
```

File: content/en-us/production/game-design/ui-ux-design.md [hunk 39]
```diff
@@ -34,20 +34,20 @@ Important information and features change during the course of a game, depending
...
 <h4>Examples</h4>
 
-Irrelevant elements can cause confusion and distraction. _Spellbound RPG_ has a clean, minimalist UI is particularly crucial for mobile interfaces, as small screens can easily get overwhelmed with excessive buttons, screens, and text.
+Irrelevant elements can cause confusion and distraction. [_Spellbound RPG_](https://www.roblox.com/games/6468323505/) has a clean, minimalist UI is particularly crucial for mobile interfaces, as small screens can easily get overwhelmed with excessive buttons, screens, and text.
 
 <GridContainer numColumns="2">
   <figure>
     <img src="../../assets/game-design/ui-ux-design/ui-ux-design-3.png" width="100%"/>
-    <figcaption>Items prior to selection in *Spellbound RPG*</figcaption>
+    <figcaption>Items prior to selection in *Spellbound RPG*</figcaption></figcaption>
```
Comment (aetter) [inline content/en-us/production/game-design/ui-ux-design.md:42]:
```suggestion
    <figcaption>Items prior to selection in *Spellbound RPG*</figcaption>
```

File: content/en-us/production/game-design/ui-ux-design.md [hunk 40]
```diff
@@ -219,7 +219,7 @@ _Metaphors_ relate an abstract concept to a more familiar one, creating a shortc
 
 <figure>
     <img src="../../assets/game-design/ui-ux-design/ui-ux-design-4.png" width="60%"/>
-    <figcaption> Spellcasting as cards in *Spellbound Wizard RPG*</figcaption>
+    <figcaption> Spellcasting as cards in [*Spellbound Wizard RPG*](https://www.roblox.com/games/6468323505/)</figcaption>
```
Comment (aetter) [inline content/en-us/production/game-design/ui-ux-design.md:222]:
```suggestion
    <figcaption> Spellcasting as cards in [*Spellbound RPG*](https://www.roblox.com/games/6468323505/)</figcaption>
```

**Review by aetter — APPROVED** [review]

**Pull request merged**